server/routes/visitors.js

```diff
@@ -0,0 +1,12 @@
+// Dependencies
+const router = require('express').Router();
+
+// Controllers
+const visitors = require('../controller/visitors');
+
+// Define routes and mapping them to controllers
```

Comment:
## Nitpick

Single line comment does not use consistent tense. You should maintain present tense, so i.e Define routes and `map` them to controllers.